### PR TITLE
Traduction de container.md

### LINKED ISSUE #1261: [FR] Localize `container` for `French`
### Please describe why this term needs to be updated

Localization of container.md into french.

### PR DESCRIPTION
**Pull request by huats**
Signed-off-by: Christophe Sauthier <christophe.sauthier@gmail.com>

### Describe your changes

This PR localizes `container.md` concept to French

### Related issue number or link (ex: `resolves #issue-number`)
resolves #1261 

### Checklist before opening this PR (put `x` in the checkboxes)
- [ X ] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [ X ] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco).

### CODE REVIEW

File: content/fr/container.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+Un conteneur est un processus qui fonctionne avec des contraintes de ressources et de capacités managées par le système d'exploitation d'un ordinateur.
+Les fichiers accessibles au processus du conteneur sont regroupés dans une image de conteneur.
+Les conteneurs sont executés les uns à coté des autres sur la même machine,
+mais habituellement le système d'exploitation empêche les processus de conteneurs différents d'interagir ensemble.
+
+## Problème qu'il adresse
+
+Avant que les conteneurs ne soient disponibles, des machines dédiées étaient nécessaires pour faire fonctionner des applications.
+Chaque machine nécessitait son propre système d'exploitation, qui utilisait du CPU, de la mémoire et de l'espace disque,
+tout ce ceci pour faire fonctionner une simple application.
```
Comment (Krast76) [inline content/fr/container.md:19]: "tout ce ceci" -> "tout cela"

File: content/fr/container.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+De plus, la maintenance, la mise à jour et le lancement d'un système d'exploitation sont des sources de travail en plus.
+
+## Quel en est l'utilité
+
+Les conteneurs partagent le même système d'exploitation et ses ressources machine,
+répartissant la surcharge en ressources liées qu système d'exploitation et permettant un usage efficace de la machine physique.
```
Comment (Krast76) [inline content/fr/container.md:25]: "qu" -> "au".

Ici on parle de "surcharge de ressource" dans le sens "les ressources disponibles" ? Je me demande si "répartissant les ressources disponibles du système d'exploitation et..."

File: content/fr/container.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+## Quel en est l'utilité
+
+Les conteneurs partagent le même système d'exploitation et ses ressources machine,
+et se répartissent la surcharge des ressources dues au système d'exploitation, permettant un usage efficace de la machine physique.
+Cette possibilité est uniquement offerte car les conteneurs sont habituellement limités dans leurs interactions mutuelles.
+Ceci permet de faire fonctionner plus d'applications sur la même machine physique.
+
+Il y a cependant des limitations.
+Etant donné que les conteneurs partagent le même système d'exploitation, les processus peuvent être considérés moins sécurisés que leurs alternatives.
```
Comment (fydrah) [inline content/fr/container.md:30]: "Étant donné..."

File: content/fr/container.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+Les conteneurs nécessitent également des limites sur les ressources partagées.
+Pour garantir les ressources, les administrateurs doivent contraindre et limiter l'utilisation de la mémoire et du CPU afin de ne pas faire baisser les performances des autres applications.
```
Comment (fydrah) [inline content/fr/container.md:32]: "...de ne pas 'impacter' les performances..." me paraîtrait plus logique.

**Review by fydrah — CHANGES_REQUESTED** [review]

File: content/fr/_TEMPLATE.md [hunk 5]
```diff
@@ -9,7 +9,7 @@ category: concept
 
 Un bref descriptif de la technologie ou du concept.
 
-## Problème qu'il adresse
+## Problème auquel il répond
 
 Quelques lignes à propos du problème qu'il adresse.
```
[the remark this answers is not in the document]
Comment (huats) [reply content/fr/_TEMPLATE.md:14]: C'est déjà changé j'ai l'impression

**Review by Krast76 — APPROVED** [review]

**Review by fydrah — APPROVED** [review]

**Pull request merged**